apps/condo/domains/news/components/NewsForm/InputStep/InputStepForm.tsx

```diff
@@ -0,0 +1,262 @@
+import { B2BAppNewsSharingConfig } from '@app/condo/schema'
+import { Col, Form, Row } from 'antd'
+import isNull from 'lodash/isNull'
+import transform from 'lodash/transform'
+import React, { useMemo } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Typography } from '@open-condo/ui'
+import { colors } from '@open-condo/ui/dist/colors'
+
+import { useLayoutContext } from '@app/condo/domains/common/components/LayoutContext'
+import { IFrame } from '@app/condo/domains/miniapp/components/IFrame'
+import { NEWS_TYPE_COMMON, NEWS_TYPE_EMERGENCY } from '@app/condo/domains/news/constants/newsTypes'
+
+import { TemplatesSelect } from '../../TemplatesSelect'
+import { getBodyTemplateChangedRule, getTitleTemplateChangedRule } from '../BaseNewsForm'
+import { TemplatesType } from '../BaseNewsForm'
+
+const NO_RESIZE_STYLE: React.CSSProperties = { resize: 'none' }
+const BIG_MARGIN_BOTTOM_STYLE: React.CSSProperties = { marginBottom: '60px' }
+const MARGIN_BOTTOM_32_STYLE: React.CSSProperties = { marginBottom: '32px' }
+const MARGIN_BOTTOM_10_STYLE: React.CSSProperties = { marginBottom: '10px' }
+const FORM_FILED_COL_PROPS = { style: { width: '100%', padding: 0, height: '44px' } }
+
+const buildCounterStyle = (textLength: number, type: 'Body' | 'Title'): React.CSSProperties => {
+    const style: React.CSSProperties = {
+        position: 'absolute',
+        right: 0,
+        margin: '12px',
+        padding: '2px 10px',
+        borderRadius: '100px',
+        backgroundColor: `${colors.gray[7]}`,
+    }
+
+    if (textLength > 0) {
+        style.backgroundColor = `${colors.black}`
+    }
+
+    if (type === 'Body') {
+        style.bottom = '12px'
+    }
+    if (type === 'Title') {
+        style.bottom = '12px'
+    }
+
+    return style
+}
+
+interface InputStepFormProps {
+    TitleInput: any
+    BodyInput: any
+    newsSharingConfig: B2BAppNewsSharingConfig
+    isSharing: boolean
+    newsItemData: {
+        type: string
+        validBefore?: string
+        title: string
+        body: string
+    }
+    selectedBody: string
+    selectedTitle: string
+    form: any
```

Comment:
`FormInstance` from `antd`.

And try to use `useForm` hook instead pass it in props:
` const [form] = Form.useForm()`